core/storage/platform_parameter/gae_models_test.py

```diff
@@ -0,0 +1,83 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Nit: 2020 (for consistency).

Reply:
Done. (Moved platform parameters back to storage/config/ as requested by another comment)